src/oauth2/controller/active-sessions.ts

```diff
@@ -20,6 +22,10 @@ class UserActiveSessions extends Controller {
       throw new Forbidden('You can only use this API for yourself yourself, or if you have \'admin\' privileges');
     }
 
+    if (!assertNotGroup(user)) {
```

Comment:
```suggestion
    if (!assertNoneGroup(user)) {
```
Wouldn't this make more sense?